src/RecursiveExtractor.Tests/RecursiveExtractor.Tests.csproj

```diff
@@ -113,4 +113,7 @@
   <ItemGroup>
     <Folder Include="TestData\" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.1.91" />
```

Comment:
I've bumped up the version here to have one version for the solution - let me know if there is any reason for RecursiveExtractor needs the older version, and I can revert this change